src/serialization/build.gradle

```diff
@@ -0,0 +1,20 @@
+dependencies {
+    compile "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
+    compile 'org.pac4j:spark-pac4j:2.0.0'
+    compile 'org.pac4j:pac4j-jwt:2.0.0'
+    compile 'commons-codec:commons-codec:1.10'
+    compile 'org.abstractj.kalium:kalium:0.6.0'
+    compile "com.opencsv:opencsv:3.9"
+    compile "com.google.code.gson:gson:2.2.4"
+    compile "com.github.salomonbrys.kotson:kotson:2.5.0"
```

Comment:
Have you confirmed that we need all of these? I'd be surprised if we needed, e.g. `kalium`.

Reply:
You're right, I can thin these down.